templates/content/en_NZ/legalities.html.twig

```diff
@@ -0,0 +1,48 @@
+{% extends 'page_base.html.twig' %}
+
+{% block content %}
+    <div class="process-box">
+        <div class="bg-light p-3 p-lg-4">
+            <h3 class="text-center font-weight-bold mb-4">
+    Legal</h3>
+
+<div class="process-details">
+    <h5 class="font-weight-bold">
+        Publisher</h5>
+
+    <ul>
+        <li>
+            Titouan Galopin : <a href="mailto:titouan@enpremiereligne.fr">titouan@enpremiereligne.fr</a>
```

Comment:
Pretty sure we are going to need to update this, but let's do it afterwards :)

Reply:
Hmm - I thought that Emmanuel had updated all of those. I know that he's still working on getting email set up for the NZ domain, so maybe that's why it's not been done.